apps/docs/src/app/core/component-docs/toolbar/examples/toolbar-example.component.ts

```diff
@@ -0,0 +1,61 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+    selector: 'fd-toolbar-type-example',
+    templateUrl: './toolbar-type-example.component.html'
+})
+export class ToolbarTypeExampleComponent implements OnInit {
+    constructor() {}
+
+    ngOnInit() {}
+}
+
+@Component({
+    selector: 'fd-toolbar-title-example',
+    templateUrl: './toolbar-title-example.component.html'
+})
+export class ToolbarTitleExampleComponent implements OnInit {
+    constructor() {}
+
+    ngOnInit() {}
+}
+
+@Component({
+    selector: 'fd-toolbar-spacer-example',
+    templateUrl: './toolbar-spacer-example.component.html'
+})
+export class ToolbarSpacerExampleComponent implements OnInit {
+    constructor() {}
+
+    ngOnInit() {}
+}
+
+@Component({
+    selector: 'fd-toolbar-separator-example',
+    templateUrl: './toolbar-separator-example.component.html'
+})
+export class ToolbarSeparatorExampleComponent implements OnInit {
+    constructor() {}
+
+    ngOnInit() {}
+}
+
+@Component({
+    selector: 'fd-toolbar-overflow-example',
+    templateUrl: './toolbar-overflow-example.component.html'
+})
+export class ToolbarOverflowExampleComponent implements OnInit {
+    constructor() {}
+
+    ngOnInit() {}
+}
+
+@Component({
+    selector: 'fd-toolbar-size-example',
+    templateUrl: './toolbar-size-example.component.html'
+})
+export class ToolbarSizeExampleComponent implements OnInit {
+    constructor() {}
+
+    ngOnInit() {}
```

Comment:
Let's remove empty constructor and ngOnInit

Reply:
done